IliasDownloaderUniMA/__init__.py

```diff
@@ -137,7 +137,16 @@ def translate_date(self, datestr):
 		"""
 
 		today = datetime.now()
-		gestern = today.replace(day = today.day-1).strftime("%d. %b %Y")
+
+		if today.day-1 == 0 & today.month-1 in [1,3,5,7,8,10,12]:
+			gestern = today.replace(day = 31).strftime("%d. %b %Y")
+		elif today.day-1 == 0 & today.month-1 not in [1,3,5,7,8,10,12]:
+			gestern = today.replace(day = 30).strftime("%d. %b %Y")
+		elif today.day-1 == 0 & today.month-1 == 2:
+			gestern = today.replace(day = 28).strftime("%d. %b %Y")
+		else:
+			gestern = today.replace(day = today.day-1).strftime("%d. %b %Y")
+			
```

Comment:
Using datetimes's [timedelta](https://docs.python.org/3/library/datetime.html#datetime.timedelta)

```suggestion
	gestern = (today - timedelta(1)).strftime("%d. %b %Y")		
```

does the same and is much cleaner IMHO.

Reply:
Thanks! I am quite new to Python and didn’t know about the timedelta function. It looks cleaner so I would also prefer your solution :-)
I quickly wrote a fix to get it to work on my machine.